src/instructlab/model/train.py

```diff
@@ -520,20 +580,340 @@ def get_files(directory: str, pattern: str) -> list[str]:
         # pull the trainrandom.randinting and torch args from the flags
         # the flags are populated from the config as a base.
         params = ctx.params
+        ds_args = DeepSpeedOptions(**params)
+        lora_args = LoraOptions(**params)
+        train_args = TrainingArgs(**params)
+        torch_args = TorchrunArgs(**params)
+        train_args.deepspeed_options = ds_args
+        train_args.lora = lora_args
+
+        if kwargs.get("e2e_training"):
+            # pulling from kwargs to not clutter function sig.
+            e2e_base_dir = kwargs.get("e2e_base_dir")
+            e2e_phase5_data = kwargs.get("e2e_phase5_data")
+            e2e_phase10_data = kwargs.get("e2e_phase10_data")
+            if all([e2e_base_dir, e2e_phase5_data, e2e_phase10_data]):
+                if click.confirm(
```

Comment:
How can we handle the non-interactive / --assume-yes case here?

Reply:
I'm not sure- that's worth investigation. Since this can be destructive (deleting existing checkpoints) I was assuming that we'd prefer to require user OK. Do you think this is required for this PR to go in?

Reply:
My opinion only: but I think it’s useful especially in the context of services building automation around rhel ai to have a non interactive path

Reply:
++ done